ext/standard/tests/strings/strip_tags_basic1.phpt

```diff
@@ -43,6 +43,7 @@ foreach($string_array as $string)
 echo "Done";
 ?>
 --EXPECT--
+Deprecated: Directive 'short_open_tag' is deprecated in Unknown on line 0
```

Comment:
For most of these tests, that ini entry doesn't really make a difference. I'd suggest to remove it from those tests instead of adding the deprecation notice.

Reply:
I left the deprecation notices in the tokenizer tests, should I remove those too?

Reply:
I'd say it is safe to remove from `token_get_all_variation15.phpt` given that the test doesn't even use the short open tags. For `002.phpt` you'd have to check if it actually influences the behavior of the tokenizer.

Reply:
Makes sense. On that PR, it would probably be more useful for that test to switch from `<?` to `<?php` to maintain its usefulness. As it stands right now, 2/3 of the things being tested are lost.

Reply:
Yeah, was probably thinking that changing the opening tag would be better than changing the whole output. Will do that when I touch on that PR again.